chsdi/models/vector/stopo.py

```diff
@@ -288,10 +299,36 @@ class LetztEisMaxMetadata (Base, ShopProductClass, Vector):
     __table_args__ = ({'schema': 'geol', 'autoload': False})
     __tablename__ = 'view_gridstand_gklgm500'
     __bodId__ = 'ch.swisstopo.gklgm500-papierkarte.metadata'
+    price = Column('price', Integer)
 
 register('ch.swisstopo.gklgm500-papierkarte.metadata', LetztEisMaxMetadata)
 
 
+class GridstandPk500(Base, ShopProductClass, Vector):
+    __tablename__ = 'view_gridstand_datenhaltung_pk500_tilecache_shop'
+    __table_args__ = ({'schema': 'datenstand', 'autoload': False})
+    __bodId__ = 'ch.swisstopo.pixelkarte-farbe-pk500.noscale'
+    price = Column('price', Integer)
+
+register('ch.swisstopo.pixelkarte-farbe-pk500.noscale', GridstandPk500)
+
+
+class Icao500Digital (Base, ShopProductClass, Vector):
+    __table_args__ = ({'schema': 'datenstand', 'autoload': False})
+    __tablename__ = 'view_gridstand_icao500_dig'
+    __bodId__ = 'ch.bazl.luftfahrtkarten-icao'
```

Comment:
price service will be ready by next week it seems
For now, we use the price in the toposhop table for papierkarten.
For fertigprodukte, the prices in the toposhop table (if they exist) refer to parameters used in the price service (probably). So for now it is not displayed in the mako (until the service is available.)
What do you think? Is it ok for you @gjn ? @p1d1d1 ?
(linked to https://github.com/geoadmin/mf-chsdi3/issues/2040)